lesson_14/class_student.py

```diff
@@ -0,0 +1,16 @@
+class Student:
+
+    def __init__(self, name, last_name, age, average_grade):
```

Comment:
For constructor it is better to provide variables types.

Reply:
Does this mean I need to write like this:
def __init__(self, name: str, last_name: str, age: int, average_grade: float) ?